.github/workflows/integration-tests.yml

```diff
@@ -19,14 +19,25 @@ jobs:
         fetch-depth: 0
     - name: Setup PR environment
       if: inputs.trigger == 'pull-request'
-      run: | # If the image with PR branch's version as tag doesn't exist, use the one in main that should already exist
+      run: | # If the image with PR branch's version as tag doesn't exist, use the one in main that should already exist (fallback on 0.4.56 incase neither exists)
         git fetch -q origin main:main
-        docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-cd-service:$(make version) > /dev/null && \
-          echo "IMAGE_TAG_CD=$(make version)" >> $GITHUB_ENV || \
+
+        if docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-cd-service:$(make version) > /dev/null; then
+          echo "IMAGE_TAG_CD=$(make version)" >> $GITHUB_ENV
+        elif docker manifest inspect ${IMAGE_REGISTRY}/kuberpult-cd-service:$(git describe --always --long --tags main) > /dev/null; then
```

Comment:
Maybe we should remove the `--long` option to get the image tag? 

```
$ git describe --always --long --tags main
0.4.56-0-gdc175cb   // which doesn't exist
```
```
$ git describe --always --tags main
0.4.56   // which is okay
```
Or do I miss something?

Reply:
😮 
Awesome!! Wasn't aware of this. love it! :)